src/solvers/base_solver.cpp

```diff
@@ -175,13 +158,14 @@ void BaseSolver::OutputState() const
   }
 
   case OutputType::GLVis: {
-    for (int i=0; i<m_state.Size(); ++i) {
+
+    for (auto & state : m_state) {
```

Comment:
I find state vs m_state a bit confusing.  Should we rename m_state to m_states? I like to clarify when my containers that hold multiple things.....

Reply:
renamed to m_states